sass-lint.yml

```diff
@@ -0,0 +1,55 @@
+#########################
+## Starter config file
+#########################
+# Linter Options
```

Comment:
Gotcha 👍 It doesn't impact our workflow since we're not running the linter as part of our build currently. I'd like to incorporate it in the future though.

Nice job on the NPM stuff. It's awesome seeing it live!